src/store/fileUploadStore.js

```diff
@@ -152,6 +164,28 @@ const actions = {
 	setAttachmentFolder(context, attachmentFolder) {
 		context.commit('setAttachmentFolder', attachmentFolder)
 	},
+
+	/**
+	 * Mark a file as shared
+	 * @param {object} context default store context;
+	 * @param {object} param1 conversation token and original file index
+	 */
+	markFileAsSharing({ commit, state }, { token, index }) {
+		if (state.files[token][index].status !== 'successUpload') {
+			throw new Error('Item is already being shared')
```

Comment:
document `@throws` or is this not common in JS?

Reply:
I don't get the question https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/throw